src/main/java/org/wikipedia/dataclient/mwapi/DepictedItem.java

```diff
@@ -0,0 +1,39 @@
+package org.wikipedia.dataclient.mwapi;
+
+import com.google.gson.annotations.SerializedName;
+
+public class DepictedItem {
+
+    @SerializedName("searchinfo")
```

Comment:
You don't need to use the annotation if the variable name is the same as the JSON key.

Reply:
OK I'll remove

Reply:
Removed in the next commit